### DISCUSSION TITLE
Optional Title when posting

### DISCUSSION THREAD

**Discussion by frank-connolly** [dailydotdev/daily#1408] [Feature Requests]
Here is a post from my squad:
![image](https://github.com/user-attachments/assets/f0447c9d-c955-4886-9830-47006ca7fe62)

We can see a relevant paragraph written by the poster about what this post is about and why he wants to share it. It's a good and relevant post. 

However, because daily dev simply uses the beginning of the description text the post is not very attractive when shown in the grid view and the chances of people checking it out are massively reduced:

![image](https://github.com/user-attachments/assets/ae72fc06-84de-4e73-b5e2-b2ad57f74e7e)

I propose an optional 'Post Title' textbox which would be used as the text shown in the post 'thumbnail'.

**Comment by idoshamun** [comment]
Hi @frank-connolly,

Thanks for the suggestion. We will consider it :)
